translations/en-us.yml

```diff
@@ -228,13 +228,18 @@ metadata:
         select: 'Select'
         tab-title-select: 'Select Schema'
         tab-title-add: 'Add'
+        saving: 'Saving Template'
     detail:
         popover:
             header: 'This is a default header'
             content: 'This is a default content'
 cedar:
     editor:
         save-button: 'Save'
+    renderer:
+        draft: 'Draft'
+        draft-explanation: "This metdata has a status of 'Draft' and is not publicly viewable. To 'Publish' this metadata please fill in all required fields and resubmit the data."
+        published-explanation: "This metdata has a status of 'Published' and is publicly viewable."
```

Comment:
Not sure if Product wants to review this language, but typo here: `metdata` -> `metadata`